events/views.py

```diff
@@ -1,3 +1,24 @@
-# from django.shortcuts import render
+from django.shortcuts import render, redirect
+from events.forms import EventCreationForm
+from django.contrib import messages
 
-# Create your views here.
+
+def create_event(request):
```

Comment:
```suggestion
@login_required
def create_event(request):
```

in my opinion, this page is only for users that have been signed in to the site

Reply:
added login_required, its redirecting to 'welcome_page' for now